maintenance/wikia/reindexImagesForWiki.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+require_once( __DIR__ . '/../Maintenance.php' );
+
+class reindexImagesForWiki extends Maintenance {
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Run ImageServingHelper::buildAndGetIndex() and clear memory cache for wiki to index images and update page_wikia_props table";
+		$this->addOption( 'wikiId', 'ID of wiki', true, true, 'w' );
```

Comment:
I don't think `wikId` should be used. This won't work correctly - `db->select` called below will load data from wiki specified by SERVER_ID, not the one from `wikiId`.
Please remove the `wikiId` param and below use `$wgServer` (and `$wgScriptPath`) instead of `wikiUrl`